requirements_test_min.txt

```diff
@@ -0,0 +1,2 @@
+-e .
+pytest~=6.2
```

Comment:
You might need to add `lazy-object-proxy`, `nose` and `wrapt` here as well. Take a look at the `tox.ini` file for that.

Reply:
Thank you for pointing that out, lazy object and wrapt are installed from the ``-e .`` but apparently we need nose and numpy.